server/src/models/database/validatable.ts

```diff
@@ -0,0 +1,8 @@
+export interface Validatable {
```

Comment:
The recipe and recipeSteps should implement this. I think wanted to still do this.